### PR TITLE
Set up document sync job for the thoth-station

### PR DESCRIPTION
**Pull request by harshad16**
Set up document sync job for the thoth-station
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Successfully ran 1st alpha test.
[example-g2p7h-document-sync-job.log](https://github.com/thoth-station/thoth-application/files/7954154/example-g2p7h-document-sync-job.log)

This is sync ceph doc from the stage(ingestion) to prod(AWS)

### CODE REVIEW

**Comment by harshad16** [comment]
cc: @fridex 
i think a daily run is more than enough i feel
if you feel the same, please review it.

File: document-sync-job/base/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              volumeMounts:
+                - name: dst-creds
+                  mountPath: /opt/app-root/src/.aws
+                  readOnly: true
+              resources:
+                requests:
+                  memory: "1Gi"
+                  cpu: "1"
+                limits:
+                  memory: "1.5Gi"
```
Comment (fridex) [inline document-sync-job/base/cronjob.yaml:86]: Might be too much - the job goes per file - I would not expect big memory consumption, but let's observe 👍🏻

File: document-sync-job/base/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: quay.io/thoth-station/document-sync-job:v0.1.0
+              name: document-sync-job
+              env:
+                - name: THOTH_DOCUMENT_SYNC_DST
+                  value: "s3://thoth-store/data/aws-prod"
```
Comment (fridex) [inline document-sync-job/base/cronjob.yaml:21]: Is this the destination bucket where prod has thoth data?

**Review by fridex — COMMENTED** [review]
Looks perfect to me 💯 👍🏻 two minor questions. Feel free to merge on your own 👍🏻

File: document-sync-job/base/cronjob.yaml (hunk 2 above)
Comment (harshad16) [reply document-sync-job/base/cronjob.yaml:21]: yes, that is correct.
`thoth` was already taken, so we had to use the `thoth-store`.

File: document-sync-job/base/cronjob.yaml (hunk 1 above)
Comment (harshad16) [reply document-sync-job/base/cronjob.yaml:86]: will bring it down.

**Comment by fridex** [comment]
/approve

👍🏻

**Pull request merged**